ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/CommandSubmitter.scala

```diff
@@ -225,24 +226,28 @@ case class CommandSubmitter(
             .map(cmds => cmds.head._1 -> cmds.map(_._2).toList)
             .buffer(maxInFlightCommands, OverflowStrategy.backpressure)
             .mapAsync(maxInFlightCommands) { case (index, commands) =>
-              timed(submitLatencyTimer, metricsManager)(
+              timed(submitLatencyTimer, metricsManager) {
                 submit(
                   id = names.commandId(index),
                   actAs = baseActAs ++ generator.nextExtraCommandSubmitters(),
                   commands = commands.flatten,
                   applicationId = generator.nextApplicationId(),
                   useSubmitAndWait = config.waitForSubmission,
                 )
-              )
+              }
                 .map(_ => index + commands.length - 1)
-                .recoverWith { case ex =>
-                  Future.failed {
+                .recoverWith {
+                  case e: io.grpc.StatusRuntimeException
```

Comment:
Can we have it enabled only for your particular style of testing?
Such that existing benchmarks are not affected.

Reply:
But all benchmarks need this, as all are affected by the rate limiting in general as you're introducing it in participant, correct?

Reply:
Correct.  The better way to do that would be to set a high limit.